File: Assignment1/assignment1.js
const fs = require('fs');

const readline = require('readline');

//Student class with grader method
class Student{
    constructor(name,assignment,grade){
        this.name =name;
        this.assignment = assignment;
        this.grade = grade;
    }
    

grader(grade){
     let lgrade;
    if(grade > 89 && grade < 101){
        lgrade = "A"
    }else if (grade > 79 && grade <90){
        lgrade = "B"
    }else if (grade > 69 && grade <80){
        lgrade = "C"
    }else if (grade > 59 && grade <60){
        lgrade = "D"
    }else if(grade > 49 && grade <1){
        lgrade = "F"
    }else if(grade > 101){
       lgrade="Invalid input, try again!"
    }else if(grade<0){
        lgrade="Invalid input, try again!"
    };
      
    return lgrade
};

};
//creating readline interface
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
})
//gathering information from the user and displaying it in the console
rl.question("What is your student's name?", (name)=>{
    rl.question("What is the assignment?", (assignment)=>{
       rl.question("What is the student's grade?",(grade)=>{
        let student = new Student(name, assignment, grade);
        console.log("Student Name: " + name )
        console.log("Assignment: " + assignment )
        console.log("Grade: " + student.grader(grade) )
         process.exit(0);
       });
      
    });
});

File: Assignment1/assignment1.py
import sys


class Grader:


    def grader(g):

        if g >= 90:
            grade = "A"
        elif g >= 80:
            grade = "B"
        elif g >= 70:
            grade = "C"
        elif g >= 60:
            grade = "D"
        elif g >= 50:
            grade = "F"
        return grade


def output(name,assignment, grade):
    if grade != 'E':
        print(name + ' got a ' + grade + ' on '+ assignment)


name=input("Enter the Student Name:") 
assignment = input("Enter Assignment Name:")
grade = int(float(input("Enter Numeric Grade:")))

g=Grader()
letter=g.grader(grade)
g.output(name,letter,assignment)

